pkg/cmd/pulumi/new.go

```diff
@@ -756,7 +756,7 @@ func chooseTemplate(templates []workspace.Template, opts display.Options) (works
 	message := "\rPlease choose a template:"
 	message = opts.Color.Colorize(colors.SpecPrompt + message + colors.Reset)
 
-	showAll := false
+	showAll := true
```

Comment:
We only set `showAll` to `true`, so if it starts as `true`, we can just omit the variable.